ken_nguyen/assignments/Ruby OOP/bankAccount.rb

```diff
@@ -0,0 +1,69 @@
+#bank account
+
+class BankAccount
+  attr_accessor :account_number, :checking, :savings
+
+  @@interest_rate = 0.06 #is it still private if its up here?
```

Comment:
Nope it's not technically private here. However, because you have no accessors it's difficult to access.